proto/nftmarket/nftmarket.proto

```diff
@@ -13,12 +13,10 @@ enum ListingType {
   LATE_SHIPPING            = 2;
 }
 enum ListingState {
-  LISTING          = 0;
-  BIDDING          = 1;
-  SELLING_DECISION = 2;
-  LIQUIDATION      = 3;
```

Comment:
Selling decision and liquidation status removed - can you confirm the change? @mkXultra